integrationtest/vert.x/pom.xml

```diff
@@ -286,16 +270,6 @@
                                     </artifact>
                                     <moduleInfo>
                                         <name>io.vertx.core</name>
-                                        <requires>
```

Comment:
Why were these customizations removed?

Reply:
Because jdeps adds them automatically so it makes a lighter config